swagger.yaml

```diff
@@ -10,49 +10,75 @@ tags:
   - name: user
     description: Operations about user
 paths:
+  /users/{userEmail}:
+    parameters:
+      - in: path
+        name: userEmail
+        schema:
+          type: string
+        required: true
+        description: Email of the user to be gotten
+    get:
+      summary: Get data about user
+      tags:
+      - user
+      description: Returns email and password hash
```

Comment:
I'd rewrite it as "Returns general info about user because it also returns id, in the future it may return something more.

Reply:
Okay, sure

Reply:
Has something changed?

Reply:
whoops